src/Parser.cpp

```diff
@@ -133,23 +130,23 @@ void PacketParser::process_packets(err_code_t& err) {
           READ_UNSIGNED(flags);
           mt_kvPtr->flags = static_cast<flags_t>(flags);
           SKIP_BYTES(1);
-          m_state = FSM_GET_FLAG;
+          m_state = FSM_GET_FLAGS;
         }
         break;
-      case FSM_GET_FLAG: // got "flag "
+      case FSM_GET_FLAGS: // got "flags "
         {
           assert(mt_kvPtr != NULL && mt_kvPtr->bytes + 1 >= mt_kvPtr->bytesRemain);
           uint64_t bytes;
           READ_UNSIGNED(bytes);
           mt_kvPtr->bytes = static_cast<uint32_t>(bytes);
-          mt_kvPtr->bytesRemain = mt_kvPtr->bytes + 1; // 1 for '\n
-          SKIP_BYTES(1);  // " " or "\r"
-          m_state = FSM_GET_BYTES_CAS;
+          mt_kvPtr->bytesRemain = mt_kvPtr->bytes + 1; // 1 for '\n'
+          SKIP_BYTES(1);  // ' ' or '\r'
+          m_state = FSM_GET_BYTES;
         }
         break;
-      case FSM_GET_BYTES_CAS: // got "bytes\r" or "bytes cas\r"
```

Comment:
state FSM_GET_BYTES_CAS means got cas，but here it's not。so split FSM_GET_BYTES_CAS to FSM_GET_BYTES and  FSM_GET_CAS。make state always means done。